completions/fish/crio.fish

```diff
@@ -109,10 +110,10 @@ complete -c crio -n '__fish_crio_no_subcommand' -l runroot -r -d 'The CRI-O stat
 complete -c crio -n '__fish_crio_no_subcommand' -f -l runtimes -r -d 'OCI runtimes, format is runtime_name:runtime_path:runtime_root:runtime_type'
 complete -c crio -n '__fish_crio_no_subcommand' -l seccomp-profile -r -d 'Path to the seccomp.json profile to be used as the runtime\'s default. If not specified, then the internal default seccomp profile will be used. (default: "")'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l seccomp-use-default-when-empty -r -d 'Use the default seccomp profile when an empty one is specified (default: false)'
-complete -c crio -n '__fish_crio_no_subcommand' -f -l selinux -d 'Enable selinux support (default: false)'
+complete -c crio -n '__fish_crio_no_subcommand' -f -l selinux -d 'Enable selinux support (default: true)'
```

Comment:
due to a limitation of the `docs-generation` and `completion` targets, it currently takes into account your crio configuration (I'm looking into it now-it's been a minor annoyance for a long time). So you'll want to do
`sed -i 's/Enable selinux support (default: true)/Enable selinux support (default: false)/g' completions/fish/crio.fish docs/crio.8.md`

and 
`sed -i 's/default: "overlay"/default: ""/g' completions/fish/crio.fish docs/crio.8.md`

I recommend sed over manually editing because I use vim, and vim spits a EOL at the end of the files which we don't currently have. You can manually edit as well, but I find doing `:set binary` and `:set noeol` annoying

Reply:
Good catch thanks, done.